hc/front/tests/test_add_check.py

```diff
@@ -11,4 +11,12 @@ def test_it_works(self):
         self.assertRedirects(r, "/checks/")
         assert Check.objects.count() == 1
 
-    ### Test that team access works
+    # Test that team access works
+    def test_team_access(self):
```

Comment:
This is testing for team access on add check. I think a better implementation would be to create a check as Alice and then login as Bob and check to see that you can see Alice's check

Reply:
This is going to be redone today

Reply:
Hello Edwin. Brian and I have just pushed a new commit after rewriting the tests